crates/language/src/language_settings.rs

```diff
@@ -443,6 +443,11 @@ pub struct IndentGuideSettings {
     /// Default: 1
     #[serde(default = "line_width")]
     pub line_width: u32,
+    /// Whether to make an active indent guide thicker than the others.
+    ///
+    /// Default: false
+    #[serde(default)]
+    pub thicken_active: bool,
```

Comment:
Let's
* keep this on by default
* update `"indent_guides"` section inside `assets/settings/default.json` with the new default and docs